src/stories/styles/Typography/Typography.stories.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import Typography from "./Typography";
+
+export default {
+  title: "Styles/Typography",
+  component: Typography,
+};
+
+const Template = (args) => <Typography {...args} />;
+
+export const Primary = Template.bind({});
+Primary.args = {
+  children: (
+    <>
+      <p className="text-browserh1 font-bold pb-4">Typography</p>
+      <p>
+        Note: Some styles change based on the size of the device. Differences
+        are specified for each individual set of styles. To see the difference
+        adjust your screen size and the "Example Text" will change accordingly
+      </p>
+      <p className="heading2 p-4">Headings</p>
```

Comment:
when I open up the storybook for this branch, I am not seeing the styles applied